docs/pages/pmd/userdocs/pmd_report_formats.md

```diff
@@ -49,10 +49,10 @@ the full rule details for each reported rule violation.
 
 Example:
 
-```
+````
```

Comment:
Is this intentional?

Reply:
I checked a few times, and Prettier kept doing that. No clue why; I assumed it knew something I don't. Happy to edit the files